### PR TITLE
Event updates

### PR DESCRIPTION
**Pull request by Aliiiu**
# Reworking Events 

This PR works on improving the user experience on the event page

## How should this be tested?

1. Got to the event page
2. Check to see if you can see events in descending order
3. Test other filters to see if it works

## Notes or observations

image

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2909

### CODE REVIEW

File: src/components/Event/EventInterest.tsx [hunk 1]
```diff
@@ -55,7 +54,7 @@ const EventInterest = ({
     const { data } = await store.dispatch(
       getEventsByTags.initiate({
         tagIds: filterTags,
-        startDate: dateFormater(new Date()),
+        // startDate: dateFormater(new Date()),
```
Comment (Royal-lobster) [inline src/components/Event/EventInterest.tsx:57]:
```suggestion
```

**Review by Royal-lobster — APPROVED** [review]

**Comment by Tolu-Mals** [comment]
The images in the slides are stretched. It becomes very apparent on mobile viewports

Screenshot

**Comment by Tolu-Mals** [comment]
the pointer should switch to a cursor when click the tags in 'interests' and 'filter'

**Comment by Tolu-Mals** [comment]
Screenshot 2024-07-24 at 10 55 01 AM

The description text is too small

**Comment by Tolu-Mals** [comment]
I feel like the description here is too small as well, maybe sync with Feyi on this

Screenshot 2024-07-24 at 10 57 53 AM

**Comment by Tolu-Mals** [comment]
the text in the tags are also really small

Screenshot 2024-07-24 at 11 00 01 AM

**Comment by Tolu-Mals** [comment]
sync with Feyi on all the design changes I suggested

**Comment by Aliiiu** [comment]
> Also, when are we implementing these guys
> 
> Screenshot 2024-07-24 at 10 51 57 AM

I have implemented them but i am not sure if it is a good user experience to see old nearby events or popular events

**Comment by Aliiiu** [comment]
> sync with Feyi on all the design changes I suggested

I feel like when you view them on your phone you won't get the same experience for the font size

**Comment by Aliiiu** [comment]
> the pointer should switch to a cursor when click the tags in 'interests' and 'filter'


https://github.com/user-attachments/assets/91b7fbec-f635-47ab-8b99-aae9bce8f644

That's the behavior. is this not what you are talking about?

**Comment by Tolu-Mals** [comment]
> > the pointer should switch to a cursor when click the tags in 'interests' and 'filter'
> 
>  Screen.Recording.2024-07-24.at.12.01.22.mov 
> That's the behavior. is this not what you are talking about?

yes. weird it's not like that on my end. ignore that then.

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
- Images are stretched
image

- Let's make "clear all" button inactive when there is no filter to clear.
- All fonts are fine.
- Text is too close with content (this is on md bp)
image
- On mobile, let's have the date and tags a little bigger. 
image

(so sorry about this: reviewed and thought i already submitted the review

**Review by Simplypheyie96 — APPROVED** [review]
Let's hide sections for where there is no data when this goes live.

**Comment by Aliiiu** [comment]
> Let's hide sections for where there is no data when this goes live.

That's what this issue solves, we want it to be populated even though the data is outdated

**Review by Tolu-Mals — APPROVED** [review]
looks good ✨

**Review by Softdev1 — APPROVED** [review]
Awesome fix

**Pull request merged**